package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -64,6 +64,20 @@
 		return v and translate('enabled') or translate('disabled')
 	end
 
+	local function formatBits(v, factor)
+		--to format a value given in kilobits use a factor of 1000
+		factor = factor or 1
+		units = {'k', 'M', 'G'}
+		units[0] = ''
+
+		bits = math.max(0, v*factor)
+		pow = math.floor(math.log(bits)/math.log(1000))
+		known_pow = math.min(pow, #units)
+
+		bits = bits/1000^known_pow
+		return string.format('%s %sbit', bits, units[known_pow])
```

Comment:
`return string.format('%**g** %sbit', bits, units[known_pow])` to make the numbers better readable.

Reply:
a73c007d7f0cd123de994b0d58c9382fa1fc4157